modules/oadp-using-legacy-aws-plugin.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-legacy-aws-plugin.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-using-legacy-aws-plugin_{context}"]
+= Using the legacy-aws Velero plugin in the DataProtectionApplication CR
+
+In the following use case, you configure the `DataProtectionApplication` CR with the `legacy-aws` Velero plugin and then back up an application.
+
+[NOTE]
+====
+Depending on the backup storage location you choose, you can use either the `legacy-aws` or the `aws` plugin in your `DataProtectionApplication` CR. Using both the plugins in the `DataProtectionApplication` CR will cause an error with the following message: `aws and legacy-aws can not be both specified in DPA spec.configuration.velero.defaultPlugins`.
```

Comment:
```suggestion
Depending on the backup storage location you choose, you can use either the `legacy-aws` or the `aws` plugin in your `DataProtectionApplication` CR. If you use both of the plugins in the `DataProtectionApplication` CR, the following error occurs: `aws and legacy-aws can not be both specified in DPA spec.configuration.velero.defaultPlugins`.
```
- Rewriting to avoid the use of future tense and to avoid a potentially ambiguous "using", per IBM Style. References:
- https://www.ibm.com/docs/en/ibm-style?topic=word-usage#using
- https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense